core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -169,7 +169,8 @@ object SparkEnv extends Logging {
       isLocal: Boolean,
       listenerBus: LiveListenerBus,
       numCores: Int,
-      mockOutputCommitCoordinator: Option[OutputCommitCoordinator] = None): SparkEnv = {
+      mockOutputCommitCoordinator: Option[OutputCommitCoordinator] = None,
+      blockManagerOption: Option[BlockManager] = None): SparkEnv = {
```

Comment:
```suggestion
      mockOutputCommitCoordinator: Option[BlockManager] = None): SparkEnv = {
```

to follow `mockOutputCommitCoordinator` above? It is also more clear that this is used as mock one in test only.

Reply:
Will do.